test/e2e/common_test.go

```diff
@@ -167,8 +150,8 @@ func printTestFailureDebugInfo(namespace string, buildRunName string) {
 
 		// Only log details of TaskRun if Tekton objects can be accessed
 		if os.Getenv(EnvVarVerifyTektonObjects) == "true" {
-			if taskRun, _ := getTaskRun(f, buildRun); taskRun != nil {
-				condition := taskRun.Status.GetCondition(apis.ConditionSucceeded)
+			if taskRun, _ := lookupTaskRun(testBuild, types.NamespacedName{Name: *buildRun.Status.LatestTaskRunRef, Namespace: namespace}); taskRun != nil {
```

Comment:
I think you need to handle `buildRun.Status.LatestTaskRunRef == nil`. The old code had no problem with that and just returned `nil` because the query in `getTaskRun` did not find something.

Reply:
Done